src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -177,12 +178,36 @@ def filter_by(
         """
         if not response_status and not metadata_filters:
             raise ValueError("At least one of `response_status` or `metadata_filters` must be provided.")
-        if response_status and not isinstance(response_status, list):
-            response_status = [response_status]
-        if metadata_filters and not isinstance(metadata_filters, list):
-            metadata_filters = [metadata_filters]
 
-        #  accessing records later
+        if response_status:
+            if not isinstance(response_status, list):
+                response_status = [response_status]
+            if not all(status in [arg.value for arg in FeedbackResponseStatusFilter] for status in response_status):
```

Comment:
I would create a constant called `_RESPONSE_STATUS_FILTER_VALUES` or something similar with value `[arg.value for arg in FeedbackResponseStatusFilter]`